### ISSUE TITLE
Sulley on Android

### ISSUE THREAD

**Issue by Xperianet** [OpenRCE/sulley#105]
Can Sulley be ported to work on Android? What requirements needed for doing that especially on process monitoring?

**Comment by Fitblip** [comment]
Yes, it should be possible to port the process monitor to android, though it would have to also do it's own crash detection/library resolution/etc. Check out https://github.com/OpenRCE/sulley/blob/master/process_monitor_unix.py
